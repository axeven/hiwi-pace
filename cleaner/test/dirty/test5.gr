p tw 9 9
1 2
1 4
2 3
3 4
4 5
5 9
6 7
6 8
7 8
