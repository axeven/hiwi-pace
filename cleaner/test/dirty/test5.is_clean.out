test5.gr dirty 9 9 not connected 0 1 7 1
